app/controllers/api_controller/logger.rb

```diff
@@ -1,5 +1,40 @@
 class ApiController
   module Logger
+    def log_request_initiated
+      @requested_at = Time.now.utc
+      api_log_info(" ")
```

Comment:
exactly, I wanted a empty log line for quick visual separator in the log file.  The split("\n") in that method does not help with an empty string, that would need to change.